portfolio/src/main/webapp/quotes.js

```diff
@@ -1,14 +1,21 @@
 /**
  * Fetch quotes from the server and appends them to the page
  */
-const addQuotesToPage = async () => {
-    const serverResponse = await fetch('/quote');
-    const responseText = await serverResponse.text();
+async function addQuotesToPage() {
+  const serverResponse = await fetch('/quote');
+  const responseJson = await serverResponse.json();
 
-    // Get the quote container and add the quote into it
-    const quoteContainer = document.querySelector('.quote-container');
-    quoteContainer.innerHTML = responseText;
-}
+  // Get a random quote from the json quotes
+  const quote =
+      responseJson
+          .quotes[Math.floor(Math.random() * responseJson.quotes.length)];
+  const quoteAuthor = quote.authorName;
+  const quoteText = quote.quoteText;
+
```

Comment:
FYI: Instead of fetching all the quotes and then selecting a random one on the client, you could have the server do that work and return just a randomly selected quotre. You don't need to make this change, I just wanted to point it out.

Reply:
Add new method to return a single quote instead of all quotes